recipes/findspark/meta.yaml

```diff
@@ -0,0 +1,34 @@
+package:
+  name: findspark
+  version: "1.0.0"
+
+source:
+  fn: findspark-1.0.0.tar.gz
+  url: https://files.pythonhosted.org/packages/b6/0b/42d86a026eb78a8d7660077fa65515ec529dce1ae85395898c977d0882f3/findspark-1.0.0.tar.gz
+  md5: 3f5eed4c63e2625070a146d980760e45
+
+build:  # noarch_python: True
```

Comment:
Please remove the comment. BTW we don't use the `noarch` option because the CIs build for all archs anyways (and that has proven to be more robust in some cases.)